code/game/objects/items/devices/PDA/PDA.dm

```diff
@@ -87,6 +88,27 @@ GLOBAL_LIST_EMPTY(PDAs)
 		inserted_item =	new /obj/item/pen(src)
 	update_icon()
 
+/obj/item/device/pda/equipped(mob/user, slot)
+	if(!equiped)
+		if(user.client)
+			switch(user.client.prefs.pda_style)
+				if(MONO)
+					font_index = 0
+					font_mode = "font-family:monospace;"
+				if(SHARE)
+					font_index = 1
+					font_mode = "font-family:\"Share Tech Mono\", monospace;letter-spacing:0px;"

```

Comment:
the names could also use the proper defines

Reply:
the names are defined but for the player facing side, this is dynamic html code